### ISSUE TITLE
Dynamis Time Remaining: 30 Second Warning

### ISSUE THREAD

**Issue by DeuceFFXI** [EdenServer/community#5691]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [ ] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

I have found evidence that mobs disappear before players are ejected from Dynamis.

Clear evidence of this can be seen in this [video](https://youtu.be/dtJI9so7SIU?t=159).  In this instance, Dynamis Lord despawns immediately upon the 30 second warning appearing in this player's chatlog.

In addition, this video of [Dynamis - Bastok](https://youtu.be/Hl9-URveXV0?t=547) it can be seen that there's still a Vanguard Drakekeeper and it's Wyvern still alive while these players are taking post boss sreenshots.  However, after the 30 second warning and just before ejection the player scans back towards where it was and it can no longer be seen.  I might concede that this is a draw distance issue for this player, but I believe the previous video suggests otherwise.

This certainly brings up an additional two questions which are related.  

1) What happens to mobs that are currently claimed by players or are active and not idle like the ones shown?
2) If they don't despawn then what would that mean for Time Extensions?

My personal recollection has vague memories of that 30 second warning being a complete end to any run.  I.e. if you got it, everything depopped regardless of where it was or what it was doing.  If I find more evidence of this mechanic I will be sure to update this report.

**Comment by DeuceFFXI** [comment]
Found additional evidence of this mechanic, which answers the two questions I posed in the original bug report.

Please refer to this [video](https://youtu.be/6ZpjAg_PoOQ?t=495).

As you can see, a player aggroes multiple mobs which were idle and pathing back to their origin points.  Other players claim a statue and soon after the 30 second warning appears in the chatlog.  At the same time as this warning appears, all mobs even those which are currently chasing a player depop instantly aside from the one which is claimed.  Players are able to kill this remaining statue and it is still able to provide drops.

Further support of this can be seen around 78:30 of this [video](https://www.nicovideo.jp/watch/sm33245709) found on NicoNico.  As in the other video the 30 second warning appears in the chatlog and the Kindred Monk remains as it is claimed.

So, in summary, once players receive the 30 second warning their run appears done.  Players are able to keep killing any claimed enemies; however any other unclaimed enemy in the zone will immediately despawn regardless of it being idle or not.

**Comment by DeuceFFXI** [comment]
Additional confirmation of this mechanic, this time showing a TE, [Voiderstreaker Butchnotch](https://youtu.be/fYOBAyf1nNA?t=163) from San d'Oria, despawning upon the 30 second warning.  These players were clearly in a rush to kill it to extend their time and failed.

I will also include this [video](https://youtu.be/QenCwHpg_0I?t=147), also from San d'Oria showing another group timing out.  This time claimed and unclaimed mobs remain in the zone after the 30 second warning.  I believe the key difference here and the video from the Bastok video in the comment above is that these mobs were actively attacking a player(s).

My guess is SE coded their logic similar to this:

30 Second Warning Trigger:
is mob idle? yes? despawn
no? is mob attacking a player/in range of their target? no? despawn

Thus, if a mob is currently attacking it will stay.  As for time extensions, I'm going to guess they fail to give a time extension even if killed after the 30 second warning appears.  It seems readily apparent that whether it's at the 3:29:30 or 2:59:30 mark this mechanic occurs.

I haven't found any evidence that killing a Time Extension after the 30 second warning is possible, but based on what I've discovered so far, logic would dictate they do not.

**Comment by DeuceFFXI** [comment]
I re-examined a [Japanese Wiki](https://ff11db.sakura.ne.jp/database/dynamis/index.html#zikan) Entry for Dynamis, which confirms my theory that killing a TE after the 30 second warning gives no extension.  In addition, they also mention that monsters without enmity will disappear confirming most of what I originally suspected in the post above.

Original Japanese text with Google Translations below: 

> 制限時間30秒前警告が出ると、ヘイトのないモンスターは消えるが、ボスを倒すと現れる???は消えずに残っている。

> When a warning appears 30 seconds before the time limit, monsters without hate will disappear, but the ??? that appears when you defeat the boss will remain.

and

> 制限時間30秒前警告が出た後、時間延長のモンスターを倒しても、時間は延長されない。

> Even if you defeat a monster that extends the time after a warning is issued 30 seconds before the time limit, the time will not be extended.


In Summary,
1. Upon the 30 Second Warning **ALL** mobs, which currently do not have a target will immediately despawn.
2. Mobs with players on their hate list should despawn once the list is empty.
3. Time Extensions killed after receipt of the 30 Second Warning will fail to extend the time remaining in Dynamis.
4. Only the ??? from killing the boss will remain until players are kicked from the zone at the time of expiration on the Perpetual Hourglass.

Therefore, so long as the Boss has players on its hate list, it should not despawn at the 30 Second Warning, and if killed prior to time expiration, players should have a short window to obtain their key item after examining the ???.

**Comment by KnowOne134** [comment]
This mechanic was added in at latter update and can be closed
